src/pages/Overview/SchoolNav/SchoolNav.jsx

```diff
@@ -1,26 +1,24 @@
 import React, { useEffect } from 'react';
 import './SchoolNav.css';
 import { Container } from 'react-bootstrap';
-import Signin from '../../../components/Login/Signin';
-import { AuthProvider, useAuthContext } from '../../../contexts/AuthContext';
+import { useAuthContext } from '../../../contexts/AuthContext';
 import logoimage from '../../../images/logo2.png';
 import { useNavigate } from 'react-router-dom';
+import SignIn from '../../../components/Login/SignIn';
 
 function LoginView(props) {
   const { school } = props; // destructuring prop or else would get linter error
   return (
     <>
-      <AuthProvider>
```

Comment:
found there were auth providers existing in code outside of the master `Provider` component, which was causing big problems as it made nested auth providerse